src/h5web/visualizations/heatmap/store.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+import { createContextStore, Action, State, action, Actions } from 'easy-peasy';
+import { ColorMap } from './interpolators';
+
+interface HeatmapState {
+  colorMap: ColorMap;
+  setColorMap: Action<HeatmapState, ColorMap>;
+}
+
+export const HeatmapStore = createContextStore<HeatmapState>({
+  colorMap: 'Magma',
+  setColorMap: action((state, colorMap) => {
+    state.colorMap = colorMap;
+  }),
+});
+
+export function useHeatmapState(): State<HeatmapState> {
+  return HeatmapStore.useStoreState(state => state);
+}
+
+export function useHeatmapActions(): Actions<HeatmapState> {
+  return HeatmapStore.useStoreActions(state => state);
+}
```

Comment:
These two hooks are just sugar. We could also use `HeatmapStore.useStoreState` and `HeatmapStore.useStoreACtions` directly in our components:

```ts
const colorMap = HeatmapStore.useStoreState(state => state.colorMap);
```